firestore/cloud-client/query_filter_or.py

```diff
@@ -12,28 +12,51 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# [START firestore_query_filter_or]
 from google.cloud import firestore
-from google.cloud.firestore_v1.base_query import FieldFilter, Or
 
 
-def query_or_composite_filter(project_id: str) -> None:
+def query_or_filter(project_id: str) -> None:
     # Instantiate the Firestore client
     client = firestore.Client(project=project_id)
-    col_ref = client.collection("users")
-
-    filter_1 = FieldFilter("birthYear", "==", 1906)
-    filter_2 = FieldFilter("birthYear", "==", 1912)
-
-    # Create the union filter of the two filters (queries)
-    or_filter = Or(filters=[filter_1, filter_2])
+    # [START firestore_query_filter_or]
+    from google.cloud.firestore_v1.base_query import FieldFilter, Or
 
+    col_ref = client.collection("cities")
     # Execute the query
-    docs = col_ref.where(filter=or_filter).stream()
+    query = col_ref.where(
+        Or(
+            FieldFilter("capital", "==", True),
+            FieldFilter("population", ">", 1_000_000),
+        )
+    )
+    docs = query.stream()
+    # [END firestore_query_filter_or]
 
     print("Documents found:")
```

Comment:
Is it intentional that the prints were left out from the sample? Someone running the sample wouldn't see anything in their screen.

Same with other samples.

Reply:
Yeah, this is in line with the other languages: https://firebase.google.com/docs/firestore/query-data/queries#or_queries. They build a query, but don't print anything from it